.github/workflows/build.yml

```diff
@@ -105,15 +105,6 @@ jobs:
           crate: grcov
           version: latest
           use-tool-cache: true
-      - name: generate coverage report
```

Comment:
If you are not uploading the coverage anymore there is no point in generating it in the first place.